validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -253,7 +276,7 @@ class Interconnect(object):
     def __init__(self,
                  endpoint,
                  dispatcher,
-                 identity=None,
+                 zmq_identity=None,
```

Comment:
Is this different from the `connection_id` that we see elsewhere?  If not, we should use the same name.